src/qt/guiutil.cpp

```diff
@@ -370,6 +371,13 @@ void bringToFront(QWidget* w)
     }
 }
 
+void handleHideShortcut(QWidget* w)
+{
+#ifdef Q_OS_MAC
+    w->connect(new QShortcut(QKeySequence(Qt::CTRL + Qt::Key_W), w), &QShortcut::activated, w, &QWidget::hide);
```

Comment:
I haven't looked closely at this,... but is there a proper deallocation of that `QShortcut`?

Reply:
I have followed the official documentation in this regard[0]. If you consider that this is not a correct application please let me know a more appropriate way.

[0] https://doc.qt.io/qt-5/qobject.html